gulpfile.js

```diff
@@ -11,6 +11,7 @@ const esperanto = require('esperanto');
 const browserify = require('browserify');
 const runSequence = require('run-sequence');
 const source = require('vinyl-source-stream');
+const header = require('gulp-header');
```

Comment:
Ah yeah.. oops.. Had a different implementation.. then realized esperanto does it.